iroh/tests/integration.rs

```diff
@@ -37,8 +39,19 @@ async fn simple_node_id_based_connection_transfer() -> Result {
     std::panic::set_hook(Box::new(console_error_panic_hook::hook));
     setup_logging();
 
-    let client = Endpoint::builder().discovery_n0().bind().await?;
+    // TODO(matheus23): Replace this with actual production relays eventually
+    let relay_map = RelayMode::Custom(RelayMap::from_iter([RelayUrl::from_str(
+        "https://philipp.iroh.link.",
+    )
+    .e()?]));
```

Comment:
@Arqu This is where an *actual* set of iterim relays would come in until we cut a release and update the canary relays